src/apis/auth.ts

```diff
@@ -0,0 +1,64 @@
+import { isAxiosError } from 'axios';
+
+import { ROUTES } from '../constants/routes';
+import { axiosInterface } from './axios';
+import { setStorage, removeStorage } from '../utils/storage';
+
+interface loginParams {
+  email: string;
+  password: string;
+}
+
+interface singupParams {
+  email: string;
+  fullName: string;
+  password: string;
+}
```

Comment:
타입에 사용되는 type이나 interface는 파스칼케이스로 작성하는게 정해진건 아니지만 규칙이라고 알고있어서 상관없을까요.?

Reply:
```suggestion
interface SignupParams {
  email: string;
  fullName: string;
  password: string;
}
```
`interface`나 `type` 같은 경우 처음에 대문자로 해야하는 것으로 알고있습니다!! 변경 부탁드립니다!!
또한 SignupParams와 SignupParams 중에 어떤 것을 해야하는지 문맥적으로 고민이네요... 어떻게 생각하시나요??
@perfumeplaylist @Jeong-Taeho

Reply:
지금 원석님이 suggestion 해주신 대로 바꾸면 좋을 것 같습니다!!

Reply:
저도 이 부분은 같이 얘기하고 싶었습니다!
지금 코드를 더 생각해서 리펙토링을 할려고 생각하고 있어서 편하게 얘기해주시면 리펙토링때 참고하겠습니다!